docs/building.rst

```diff
@@ -524,7 +531,8 @@ build configurations
 By default libtorrent is built In debug mode, and will have pretty expensive
 invariant checks and asserts built into it. If you want to disable such checks
 (you want to do that in a release build) you can see the table below for which
-defines you can use to control the build.
+defines you can use to control the build. Make sure to define the same macros in your
+own code that compiles and links with bittorrent.
```

Comment:
I think you meant to say "libtorrent" here, not "bittorrent"

Reply:
Yes, sorry about this.